.pre-commit-config.yaml

```diff
@@ -92,6 +92,12 @@ repos:
         args: ["--toml", "pyproject.toml"]
         additional_dependencies: [tomli]
 
+  - repo: https://github.com/macisamuele/language-formatters-pre-commit-hooks
+    rev: v2.10.0
+    hooks:
+    - id: pretty-format-toml
+      args: [--autofix, --indent, '2']
```

Comment:
Any particular reason why 2 spaces instead of 4 ? I can't find a good, accepted [style guide for TOML](https://github.com/toml-lang/toml/issues/608), so I'd think this is just developer preference.

Reply:
I honestly went with what was in the read of the  pre-commit hook repo? You want 4, let's do 4.you want 8, let's do 8. You want 16... Get me a larger screen.

Reply:
😂 Just wanted to confirm that there wasn't a style guide I was missing ! I'm fine with 2 or 4.

Reply:
given that our code uses 4, let's go with 4 to avoid the mental gymnastic of 4 for python, 2 for toml (my brain is too old and unflexible for those exercises)